tests/services/desk_handler_service_test.py

```diff
@@ -1,19 +1,22 @@
 from vcr_unittest import VCRTestCase
+from mock import patch
 from services.desk_handler_service import DeskHandlerService
 from config.config import CONFIG
 
 class TestDeskHandlerService(VCRTestCase):
-    def test_create_is_successful(self):
+
+    @patch("config.config.__YAMLConfigLoader._YAMLConfigLoader__write_to_config", return_value = None)
```

Comment:
Had to patch `_YAMLConfigLoader.__write_to_config` because it was actually writing to `config/config.yaml` on every test run.